dev-docs/bidders/greenbids.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: bidder
+title: Greenbids
+description: Prebid Greenbids Bidder Adapter
+biddercode: greenbids
+tcfeu_supported: true
```

Comment:
```suggestion
tcfeu_supported: false
```

if you have no `gvl_id` you cannot claim `tcfeu_supported`

Reply:
@jeremy-greenbids this needs to change as well, then this is ready to be merged, once the prebid adapter is released.

Reply:
Done, sorry